### ISSUE TITLE
Cisco Spark Disabled TLS 1.0

### ISSUE THREAD

**Issue by swapinfutures** [darrenparkinson/SparkDotNet#12]
Cisco Spark has disabled TLS 1.0 which has broken this library:

https://developer.ciscospark.com/blog/blog-details-9497.html

**Comment by darrenparkinson** [comment]
Thanks for logging the issue. Are you able to provide any details about your environment such as version of dotnet etc? I should have time to look into it tomorrow — it doesn’t seem like this should require too much change.

**Comment by darrenparkinson** [comment]
Hi, I've performed some testing this morning without changing anything and I don't appear to be having any trouble?  Could you provide some more detail about your exact issue so I could look into it?  Many thanks.  D.

**Comment by jnallu** [comment]
Hello,
I had this issue on 17 of this month. I had to update my services/apps to support TLS1.1 .
I included this line of code before make call to the Spark API using the library 
"  ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;" .
We should be using .netframework 4.5 or above.
I have read somewhere that If we use framework 4.6 TLS 1.1 isuspported by default but I haven't tested it.
